tests/service/test_service_disable.yml

```diff
@@ -40,7 +40,7 @@
       - "{{ lookup('file', 'cert1.b64', rstrip=False) }}"
       force: no
     register: result
-    failed_when: not result.changed
+    failed_when: not result.changed or result.failed
```

Comment:
This change is unrelated.

Reply:
This change is in a TEMP commit, so service tests pass.